kcbq-connector/src/test/java/com/wepay/kafka/connect/bigquery/write/row/BigQueryWriterTest.java

```diff
@@ -275,7 +275,7 @@ public void testBigQueryCompleteFailure() {
     testTask.put(sinkRecordList);
     Exception expectedEx =  assertThrows(BigQueryConnectException.class, 
                                         () -> testTask.flush(Collections.emptyMap())); 
-    assertTrue(expectedEx.getMessage().contains("test_topic"));
+    assertTrue(expectedEx.getCause().getMessage().contains("test_topic"));
```

Comment:
This test was failing as the expected exception was wrapped by BigQueryConnectException. Made this change for all test cases to pass.